package.json

```diff
@@ -10,6 +10,7 @@
     "lint": "turbo run lint",
     "fix": "turbo run fix",
     "test": "turbo run test --no-cache --concurrency=1",
+    "test:integration": "turbo run test:integration --no-cache --concurrency=1",
```

Comment:
why do you limit the concurrency of 1? If you don't have a limit, you can run fast.

Reply:
Fixed